--- /home/davidc/dev/mayaMatchMoveSolver/external/archives/levmar-2.6/levmar.h	2011-11-30 18:37:45.000000000 +1030
+++ /home/davidc/dev/mayaMatchMoveSolver/external/working/levmar-2.6/levmar.h	2017-12-13 05:13:55.099562046 +1030
@@ -28,7 +28,7 @@
  */
 
 /* specifies whether to use LAPACK or not. Using LAPACK is strongly recommended */
-#define HAVE_LAPACK
+// #define HAVE_LAPACK
 
 /* specifies whether the PLASMA parallel library for multicore CPUs is available */
 /* #undef HAVE_PLASMA */
