--- /home/davidc/dev/mayaMatchMoveSolver/external/archives/levmar-2.6/Makefile	2011-11-28 22:36:00.000000000 +1030
+++ /home/davidc/dev/mayaMatchMoveSolver/external/working/levmar-2.6/Makefile	2017-12-23 19:46:25.670882122 +1030
@@ -6,8 +6,7 @@
 CC=gcc
 CONFIGFLAGS=#-ULINSOLVERS_RETAIN_MEMORY
 #ARCHFLAGS=-march=pentium4 # YOU MIGHT WANT TO UNCOMMENT THIS FOR P4
-CFLAGS=$(CONFIGFLAGS) $(ARCHFLAGS) -O3 -funroll-loops -Wall #-g #-ffast-math #-pg
-LAPACKLIBS_PATH=/usr/local/lib # WHEN USING LAPACK, CHANGE THIS TO WHERE YOUR COMPILED LIBS ARE!
+CFLAGS=$(CONFIGFLAGS) $(ARCHFLAGS) -I$(LAPACK_INCLUDE_DIR) -O3 -funroll-loops -Wall -m64 -fPIC #-g #-ffast-math #-pg
 LDFLAGS=-L$(LAPACKLIBS_PATH) -L.
 LIBOBJS=lm.o Axb.o misc.o lmlec.o lmbc.o lmblec.o lmbleic.o
 LIBSRCS=lm.c Axb.c misc.c lmlec.c lmbc.c lmblec.c lmbleic.c
@@ -15,20 +14,7 @@
 DEMOSRCS=lmdemo.c
 AR=ar
 RANLIB=ranlib
-LAPACKLIBS=-llapack -lblas -lf2c # comment this line if you are not using LAPACK.
-                                 # On systems with a FORTRAN (not f2c'ed) version of LAPACK, -lf2c is
-                                 # not necessary; on others, -lf2c is equivalent to -lF77 -lI77
-
-#LAPACKLIBS=-L/usr/local/atlas/lib -llapack -lcblas -lf77blas -latlas -lf2c # This works with the ATLAS updated lapack and Linux_P4SSE2
-                                                                            # from http://www.netlib.org/atlas/archives/linux/
-
-#LAPACKLIBS=-llapack -lgoto2 -lpthread -lf2c # This works with GotoBLAS
-                                             # from http://www.tacc.utexas.edu/research-development/tacc-projects/
-
-#LAPACKLIBS=-L/opt/intel/mkl/8.0.1/lib/32/ -lmkl_lapack -lmkl_ia32 -lguide -lf2c # This works with MKL 8.0.1 from
-                                            # http://www.intel.com/cd/software/products/asmo-na/eng/perflib/mkl/index.htm
-                                            # check the MKL link line advisor for more information:
-                                            # http://software.intel.com/en-us/articles/intel-mkl-link-line-advisor/
+LAPACKLIBS=-L$(LAPACKLIBS_PATH) -llapack -lcblas -lf77blas -latlas
 
 LIBS=$(LAPACKLIBS)
 
