--- ./external/archives/sparselm-1.3/Makefile	2011-12-05 06:41:52.000000000 +1030
+++ ./external/working/sparselm-1.3/Makefile	2017-12-12 19:35:51.233351059 +1030
@@ -3,9 +3,9 @@
 #
 
 CC=gcc
-GCCFLAGS=-O3 -Wall -ffast-math -funroll-all-loops #-g #-fomit-frame-pointer # -msse3 # -fprofile-arcs -ftest-coverage # -pg # GCC-specific optimization flags
-MACH_CFLAGS=-march=pentiumpro -malign-double # machine specific gcc flags, in this case for the pentium pro
-#MACH_CFLAGS=-march=native -mtune=native
+GCCFLAGS=-O3 -Wall -ffast-math -funroll-all-loops -fPIC -msse3 #-g #-fomit-frame-pointer # -msse3 # -fprofile-arcs -ftest-coverage # -pg # GCC-specific optimization flags
+# MACH_CFLAGS=-march=pentiumpro -malign-double # machine specific gcc flags, in this case for the pentium pro
+MACH_CFLAGS=-m64
 
 SRCS=splm.c splm_crsm.c splm_ccsm.c splm_hessian.c splm_time.c splm_stm.c splm_misc.c
 OBJS=splm.o splm_crsm.o splm_ccsm.o splm_hessian.o splm_time.o splm_stm.o splm_misc.o
@@ -21,20 +21,21 @@
 include solvers.mk
 
 ## header file paths
-SUITESPARSE_PATH=/home/lourakis/SuiteSparse-3.4.0
+SUITESPARSE_PATH=../SuiteSparse
+MKLROOT=/opt/intel/mkl
 
-CSPARSE_INC=-I$(SUITESPARSE_PATH)/CSparse/Include
-LDL_INC=-I$(SUITESPARSE_PATH)/LDL/Include
-CHOLMOD_INC=-I$(SUITESPARSE_PATH)/CHOLMOD/Include
+CSPARSE_INC=-I$(SUITESPARSE_PATH)/include
+LDL_INC=-I$(SUITESPARSE_PATH)/include
+CHOLMOD_INC=-I$(SUITESPARSE_PATH)/include
 PARDISO_INC=
-UMFPACK_INC=-I$(SUITESPARSE_PATH)/UMFPACK/Include
+UMFPACK_INC=-I$(SUITESPARSE_PATH)/include
 SuperLU_INC=-I/usr/local/include/slu
 TAUCS_INC=-I/usr/local/include/
 SPOOLES_INC=-I/home/lourakis/spooles.2.2/
 MUMPS_INC=-I/home/lourakis/MUMPS_4.8.3/include/ -I/home/lourakis/MUMPS_4.8.3/libseq
-MKL_INC=-I/opt/intel/mkl/8.0.1/include/
-UFconfig_INC=-I$(SUITESPARSE_PATH)/UFconfig/
-AMD_INC=-I$(SUITESPARSE_PATH)/AMD/Include/
+MKL_INC=-I${MKLROOT}/include/
+UFconfig_INC=-I$(SUITESPARSE_PATH)/include/
+AMD_INC=-I$(SUITESPARSE_PATH)/include/
 METIS_INC=
 
 ######## No changes should be necessary beyond this point ########
